syft/frameworks/torch/tensors/interpreters/replicated_shared.py

```diff
@@ -270,6 +281,128 @@ def shape(self):
     def players(self):
         return self.__get_players()
 
+    def set_garbage_collect_data(self, value):
+        shares = self.child
+
+        for _, shares in shares.items():
+            for share in shares:
+                assert share.is_wrapper is True
+                share.gc = value
+
+    def get_garbage_collect_data(self):
+        shares = self.child
+        res = None
+
+        """ Select the first share """
```

Comment:
Docstring should only be at the begining of a function

Reply:
Removed :D